tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderingPartyControllerTests.cs

```diff
@@ -147,7 +147,7 @@ private OrderingPartyTestContext()
 
             internal Guid PrimaryOrganisationId { get; }
 
-            internal ClaimsPrincipal ClaimsPrincipal { get; }
+            private ClaimsPrincipal ClaimsPrincipal { get; }

```

Comment:
Not essential, but now that this is `private` ideally it should be below the `internal` properties (based on ordering from most accessible > least accessible).

Reply:
👍